src/trio/_core/_run.py

```diff
@@ -1871,7 +1889,6 @@ async def python_wrapper(orig_coro: Awaitable[RetT]) -> RetT:
 
             coro = python_wrapper(coro)
```

Comment:
I think this isn't needed now that we don't use cr_frame, but I know that various things expect trio.lowlevel.current_task().cr_await and trio.lowlevel.current_task().cr_frame to work so I'm leaving it here